curriculum/challenges/english/01-responsive-web-design/accessibility-quiz/part-022.md

```diff
@@ -0,0 +1,127 @@
+---
+id: 6143920c8eafb00b735746ce
+title: Part 22
+challengeType: 0
+dashedName: part-22
+---
+
+# --description--
+
+Even though you added a `placeholder` to the first `input` element in the previous lesson, this is actually not a best-practice for accessibility; too often, users confuse the placeholder text with an actual input value - they think there is already a value in the input.
+
+Remove the placeholder text from the first `input` element, relying on the `label` being the best-practice.
+
+# --hints--
+
+You should remove the `placeholder` attribute from the first `input` element.
+
+```js
+assert.isEmpty(document.querySelectorAll('input')?.[0].getAttribute('placeholder'));
```

Comment:
This returns `null` for me, which doesn't seem to pass `isEmpty`.

Reply:
This is why you should not use `getAttribute` unless you understand what the difference is.